services/wallet/token/token.go

```diff
@@ -107,6 +114,31 @@ func (tm *Manager) inStore(address common.Address, chainID uint64) bool {
 	return ok
 }
 
+func (tm *Manager) areTokensFetched() bool {
+	for _, store := range tm.stores {
+		if !store.areTokensFetched() {
+			return false
+		}
+	}
+
+	return true
+}
+
+func (tm *Manager) fetchTokens() []*Token {
+	result := make([]*Token, 0)
+	for _, store := range tm.stores {
+		tokens, err := store.GetTokens()
+		if err != nil {
+			log.Error("can't fetch tokens from store: %s", err)
```

Comment:
@IvanBelyakoff you're not fetching them, the list of tokens is hardcoded, according to my previous comment you don't need this `if err != nil {` check.